forge-gui/res/cardsfolder/upcoming/the_brothers_war.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Brothers' War
+ManaCost:3 R
+Types:Enchantment Saga
+K:Saga:3:DBPowerStone,DBChoose,DBDamage
+SVar:DBPowerStone:DB$ Token | TokenAmount$ 2 | TokenTapped$ True | TokenScript$ c_a_powerstone | TokenOwner$ You | SpellDescription$ Create two tapped Powerstone tokens.
+SVar:DBChoose:DB$ ChoosePlayer | DefinedPlayer$ Targeted | TargetMin$ 2 | TargetMax$ 2 | ValidTgts$ Player | Choices$ ThisTargetedPlayer | AILogic$ Curse | SubAbility$ DBEffect | SpellDescription$ Choose two target players. Until your next turn, each creature they control attacks the other chosen player each combat if able.
```

Comment:
I think you can just target players on an Effect line, so you won't need to deal with ChoosePlayer

Then you won't need a Cleanup for ChosenPlayer either

Reply:
Some background info for you:

"Choose two target players" is just how they template it for easier reading. But we know from the rules there's no replacement or trigger like "Whenever you choose a player..." so you have more freedom there. Mind you some effect API would behave differently if you did Targeting on them so that's not always possible